src/main/resources/api/service_openapi.yaml

```diff
@@ -68,6 +68,55 @@ paths:
         500:
           description: Get request error
           $ref: '#/components/responses/ErrorResponse'
+  '/api/v1/workspaces/{id}/datareferences':
+    parameters:
+      - $ref: '#/components/parameters/Id'
+    get:
+      description: |
+        Enumerate the data references in a workspace.
+      operationId: enumerateReferences
+      tags:
+      - workspace
+      parameters:
+      - in: query
+        name: offset
+        description: The number of items to skip before starting to collect the result set.
+        schema:
+          type: integer
+          default: 0
+      - in: query
+        name: limit
+        description: The numbers of items to return. Default 10
+        schema:
+          type: integer
+          default: 10
+      - in: query
+        name: filterControlled
+        description: |
+          Filter for selecting controlled vs. uncontrolled resources.
+        schema:
+          type: string
+          default: all
+          enum:
+          - controlled
```

Comment:
This should make an Enum that you can use elsewhere.

Reply:
TIL OpenAPI doesn't generate the enum class if you define it in-line with parameters. pulled this out into the schema section and wired it through.